### PR TITLE
[libxslt] update to cmake

### PR DESCRIPTION
**Pull request by Neumann-A**

### CODE REVIEW

**Review by Adela0814 — APPROVED** [review]

File: ports/libxslt/portfile.cmake [hunk 1]
```diff
@@ -202,3 +80,4 @@ file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 
 file(INSTALL "${SOURCE_PATH}/Copyright" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+file(COPY "${CMAKE_CURRENT_LIST_DIR}/vcpkg-cmake-wrapper.cmake" DESTINATION "${CURRENT_PACKAGES_DIR}/share/libxslt")
```
Comment (Thomas1664) [inline ports/libxslt/portfile.cmake:83]: Can you add a final newline?

Comment (BillyONeal) [reply ports/libxslt/portfile.cmake]: Fixed.

**Review by BillyONeal — APPROVED** [review]
CMake all the things woo!

**Comment by BillyONeal** [comment]
Thanks again!

**Pull request merged**